### PR TITLE
Bugfix/rendering order

### LINKED ISSUE #102: Bullets go behind walls
Bullets from the player and enemies go behind the background walls in the levels where they are invisible. They should be in the same layer as the characters

### PR DESCRIPTION
**Pull request by WillTarte**
Fixes #102 & #109

### CODE REVIEW

**Comment by WillTarte** [comment]
I had to start off from Édouard's branch, so maybe we'll wait till it gets merged?

**Comment by tedgag** [comment]
The bug happens for grenades as well. Also make sure that the bug is fixed for level 2 before merging

**Review by Wmbat — APPROVED** [review]
Looks good

**Pull request merged**
